.gitignore

```diff
@@ -8,12 +8,16 @@ __pycache__
 
 /graveyard-venv
 /gvenv
-/.vscode
 /.directory
 
 /graveyard/settings/local.py
 /graveyard/settings/production.py
 
 .DS_store
 
+# Local building tools
 /docker-compose.yml
+
+## Local IDEs dirs
+.idea
```

Comment:
It's always in the root folder—how about `/.idea`? 

Plus what I do is to have it in the global .gitignore :)

Reply:
I thought someone might open accidentally the folder `views` and then it could be pushed non intentionally.

I'm not sure if I follow the global .gitignore. This is the main gitignore for the project, isn't it?

Reply:
You can have gitignore that is valid for all projects:

```
almad@hyuga /tmp % cat ~/.gitconfig 
[core]
	excludesFile = ~/.gitignore
almad@hyuga /tmp % cat ~/.gitignore 
/.vscode
/.idea
.DS_Store

__pycache__

/venv

almad@hyuga /tmp % 
```

Reply:
Ah, this will be some more learning for me. But for now I'd let it in, just to be sure. :)